# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/spirv_cross/.cargo-checksum.json
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/spirv_cross/.cargo-checksum.json b/third_party/rust/spirv_cross/.cargo-checksum.json
deleted file mode 100644
--- a/third_party/rust/spirv_cross/.cargo-checksum.json
+++ /dev/null
@@ -1,1 +0,0 @@
-{"files":{"Cargo.toml":"5a4a2dfbf986e715feebebc5b5601af1fee9f1c6d0cfc0f91ce00754981133e2","README.md":"fe004c99185268fa531fd378778d17ed3b7d8d9e63b5c756a66687a20f459f9e","lib.rs":"ca2f0e92795ee0e5b905b47dbc200e5c3bce0da9edef06b0313b8c3f03c2fcf5"},"package":null}
\ No newline at end of file